test-adoc.sh

```diff
@@ -737,6 +738,23 @@ function test_module_headings {
 }
 
 
+# Verifies that there are no dangerous  yum install -y commands.
+#
+# Usage: test_yum_install_y FILE
+function test_yum_install_y {
+  local -r filename="$1"
+
+  (print_adoc "$filename" | grep -qP "yum[\s]+install[\s]+-y")
```

Comment:
Thank you very much for implementing this, @VladimirSlavik! There are, however, two ways this option is commonly used (and [both are present in RHEL documentation](https://www.google.com/search?client=firefox-b-d&q=site%3Aaccess.redhat.com%2Fdocumentation+%22yum+-y+install%22)):

    yum install -y
    yum -y install

Would you mind tweaking the test to discover both?

Reply:
Ah, you're right. And now that I think about it, actually the whole premise of yum -y is flawed, whether it is install, or update, or distro-sync... So it's in fact appearance of yum(...)-y that is bonkers.